src/torchcodec/_core/Encoder.h

```diff
@@ -26,12 +26,14 @@ class AudioEncoder {
  private:
   void encodeInnerLoop(
       AutoAVPacket& autoAVPacket,
-      const UniqueAVFrame& avFrame);
+      const UniqueAVFrame& srcAVFrame);
   void flushBuffers();
+  AVSampleFormat findOutputSampleFormat(const AVCodec& avCodec);
```

Comment:
maybe this shouldn't be a method and just a function in a private namespace?

Reply:
If it doesn't use any class members, then, yeah, let's make it a function in anonymous namespace.